pallets/proposals/src/impls.rs

```diff
@@ -206,7 +209,11 @@ impl<T: Config> Pallet<T> {
         let expiry_block = frame_system::Pallet::<T>::block_number()
             .saturating_add(<T as Config>::NoConfidenceTimeLimit::get());
 
-        Rounds::<T>::insert(project_key, RoundType::VoteOfNoConfidence, expiry_block);
+        Rounds::<T>::insert(
+            (project_key, 0),
```

Comment:
What happens if the round of no confidence is raised after milestone 0 has been approved?

Reply:
They're indépendant of each other due to the use of the round type key so we will be all good there :)